data/avro-schema/src/main/resources/avro/net/corda/data/certificates/rpc/request/CertificateRpcRequest.avsc

```diff
@@ -5,9 +5,9 @@
   "doc": "RPC request regarding certificates.",
   "fields": [
     {
-      "name": "tenantId",
-      "doc": "'p2p', 'rpc-api', or holding identity identity ID.",
-      "type": "string"
+      "name": "usage",
+      "doc": "The certificate usage. Either CertificateType or holding identity ID.",
+      "type": ["net.corda.data.certificates.CertificateType", "string"]
```

Comment:
Seems like we have two different pieces of information (the certificate type and whether the corresponding key is a cluster-level or vnode-level one) mussed up into one field/type here? Wouldn't it be cleaner to have separate fields to model each thing?

Reply:
Certificate can either be a cluster level certificate (with a type) or a virtual node certificate.

Reply:
but, isn't the concept of a certificate type useful generally regardless of whether it's a cluster-evel or vnode-level one i.e. shouldn't we keep hold of that information for vnode-level certificates too?

Reply:
(what I was thinking is having a separate field that indicates the type of the certificate and a separate field that indicates whether it's a cluster-level or a vnode-level one)